job-service/docs/pages/en-us/Getting-Started-batchworker.md

```diff
@@ -91,81 +91,184 @@ It is the responsibility of the Batch Worker plugin to repeatedly call the Batch
         }
     }
 
-#### Task Message Builder
-The `processBatch()` method of the Batch Worker Plugin can use the CAF ModuleProvider to load the appropriate task message builder implementation. The builder implementation can then be used to create a worker specific task message for each individual worker item using the `buildMessage()` method. In the sample code below, the `buildMessage()` method of the Document Message Builder requires a `DocumentServices` object that can be used to retrieve a `Document` object using the storage reference specified in the batch definition. It also requires additional message parameter details for the target worker.
+### Batch Worker Container module
 
-    @Override
-    public void processBatch(BatchWorkerServices batchWorkerServices,
-                             String batchDefinition,
-                             String taskMessageType,
-                             Map<String, String> taskMessageParams)
-        throws BatchDefinitionException
-    {
-        ...
-        
-        if (isSubBatchTrue) {
-            ...
-        } else {
-            // Use the CAF ModuleProvider to load the appropriate task message
-            // builder implementation.
-            DocumentMessageBuilder builder = ModuleProvider.getInstance()
-                .getModule(DocumentMessageBuilder.class, taskMessageType);
-            ...
-            
-            // Construct the DocumentServices object
-            Document document = new DocumentImpl(reference);
-            DocumentServices documentServices = new DocumentServicesImpl(document);
-            
-            // Create worker specific task message for individual worker item
-            TaskMessage taskMessageToSend =
-                builder.buildMessage(documentServices, taskMessageParams);
-            ...
-            
-            // Register individual worker item
-            batchWorkerServices.registerItemSubtask(...);
-        }
-    }
+The Batch Worker Container module of the project consumes the Batch Worker Plugin implementation as a dependency, produces an a container from with it and runs testcase integration testing against the container with the `BatchWorkerAcceptanceIT.java` class.
 
-In order to utilise the CAF ModuleProvider, you will need to reference the [util-moduleloader](https://github.hpe.com/caf/caf-common/tree/develop/util-moduleloader) artifact in the project POM:
+The next section, [Creating a Batch Worker from Archetype](#Creating-a-Batch-Worker-from-Archetype), will create a basic Batch Worker project for you that contains both Batch Worker Plugin and Batch Worker Container modules. It is recommended that you use this as the basis of your Batch Worker.
 
-    <dependency>
-        <groupId>com.github.cafapi.util</groupId>
-        <artifactId>util-moduleloader</artifactId>
-        <version>1.0.0</version>
-    </dependency>
+## Creating a Batch Worker from Archetype
```

Comment:
As I proposed above rename to 'Creating a Batch Worker from a Maven Archetype'. Include some text and a link of to a separate guide on archetype usage instructions. I'm only suggesting this to minimise the size of this getting started guide as well as avoid duplication of usage instructions in multiple projects.